Module-7-Assignment/Question-7/ManageClassController.cls

```diff
@@ -0,0 +1,55 @@
+/**
+ * Controller for page ManageClass.
+ */ 
+public class ManageClassController 
```

Comment:
Here user is being redirected to the same page whenever save or delete section occurs.
The question requires to do the following instead
1) Open the edit section with the relevant class in editable mode whenever user clicks on the edit link at the bottom of the page
2) When user updates the class in the edit section and clicks on save button then close the edit section and reRender the class list with the updated details about the class that was updated
The use should remain on the same page

Reply:
Done